geniuspaste/data/tinypaste.com.conf

```diff
@@ -0,0 +1,20 @@
+# FIXME: outdated, this leads to a 404 on http://pasted.co/
```

Comment:
So in the end, it's merely ported from the previous hard-coded logic, with zero testing :)